pkg/cli/resources.go

```diff
@@ -196,7 +196,12 @@ func ResourcesProxy(rack sdk.Interface, c *stdcli.Context) error {
 		return fmt.Errorf("no url for resource: %s", r.Name)
 	}
 
-	u, err := url.Parse(r.Url)
+	resourceUrl := r.Url
+	if !strings.Contains(r.Url, "http") {
+		resourceUrl = fmt.Sprintf("http://%s", r.Url)
```

Comment:
should it be https instead?

Reply:
it won't matter if it's not HTTP or HTTPS, the problem was that URL parsing expects a scheme and if doesn't have one then fails to parse the port. The important thing is the port.